src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -1368,6 +1368,11 @@ type SuspensionStatusHistory {
   then the suspension has not been cancelled/edited.
   """
   modifiedAt: Time
+
+  """
+  message is an optional custom message to the suspended user.
+  """
+  message: String
```

Comment:
actually now I'm wondering if this should be mandatory... there's no fallback on the server side. The default text is provided by the client, if no message gets passed through you'd get a mostly blank email.

Reply:
Agreed!